test-app/package.json

```diff
@@ -47,7 +47,7 @@
     "ember-cli-babel": "^8.2.0",
     "ember-cli-htmlbars": "^6.3.0",
     "ember-cli-inject-live-reload": "^2.1.0",
-    "ember-data": "~3.28.13",
+    "ember-data": "~5.3",
```

Comment:
test-app now running tests against 5.3

If it was easy as adding a try scenario for ED 3 and 4.12 I would, but the tests need to be different for each, too many things changed